### PR TITLE
Expansion Hub module

### PR DESCRIPTION
**Pull request by LandryNorris**
Created a high level module for expansion hub.

### CODE REVIEW

**Comment by LandryNorris** [comment]
Depends on #37

**Comment by LandryNorris** [comment]
Renamed the old 'RevHub' interface to 'ExpansionHub' for better future-proofing.

File: packages/expansion-hub/src/open-rev-hub.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+/**
+ * Maps the serial port path (/dev/tty1 or COM3 for example) to an open
+ * Serial object at that path. The {@link Serial} object should be removed from
+ * the map upon closing.
+ */
+const openSerialMap = new Map<string, Serial>();
+
+/**
+ * Opens a parent REV hub, given that you know its {@link serialNumber} (should start with DQ).
+ * Any children of this hub will be automatically discovered and initialized.
```
Comment (NoahAndrews) [inline packages/expansion-hub/src/open-rev-hub.ts:15]: This sentence is false. Discovery is only performed to find the parent module

File: packages/rhsplib/tsconfig.json
Comment (NoahAndrews) [inline packages/rhsplib/tsconfig.json:1]: This should only extend the top-level config.

File: packages/expansion-hub/tsconfig.json
Comment (NoahAndrews) [inline packages/expansion-hub/tsconfig.json:1]: This should only extend the top-level config.

**Review by NoahAndrews — CHANGES_REQUESTED** [review]

File: packages/rhsplib/tsconfig.json
Comment (LandryNorris) [reply packages/rhsplib/tsconfig.json:1]: Changed this

File: packages/expansion-hub/tsconfig.json
Comment (LandryNorris) [reply packages/expansion-hub/tsconfig.json:1]: Changed this

**Review by NoahAndrews — APPROVED** [review]

**Pull request merged**